src/emailpro/account/emailpro-account.controller.js

```diff
@@ -183,10 +184,23 @@ angular.module('Module.emailpro.controllers').controller('EmailProTabAccountsCtr
 
   $scope.addNewConfigureAccount = function () {
     $scope.loadingNewConfiguredAccount = true;
-    EmailPro.getAccounts().then((accounts) => {
-      const newConfiguredAccount = _.find(accounts.list.results, account => /.*configureme\.me$/.test(account.domain));
-      $scope.setAction('emailpro/account/update/emailpro-account-update', newConfiguredAccount);
-    }).catch(err => $scope.setMessage($translate.instant('emailpro_tab_ACCOUNTS_error_message'), err))
+    const newConfiguredAccount = _.find($scope.accountIds, (account) => {
+      const [, domain] = account.split('@');
+      return /.*configureme\.me$/.test(domain);
+    });
+    EmailPro.getAccount({
```

Comment:
`return` ?

Reply:
You're right! Since the hotfix has been validated it could be addressed in a separate pull request.

Thanks for pointing out this!